Client/Frontend/Home/BookmarksPanel.swift

```diff
@@ -124,11 +172,16 @@ class BookmarksPanel: SiteTableViewController, HomePanel {
     }
 
```

Comment:
I saw your IRC comment about the bookmark cell still moves to the side for default bookmarks even though there's no edit actions. Instead of setting the editActions below to an empty array when source = nil, you can implement the editingStyle delegate method that tells the tableView if it should edit or not. This will remove that slight jiggle for default bookmarks:

```swift
    func tableView(tableView: UITableView, editingStyleForRowAtIndexPath indexPath: NSIndexPath) -> UITableViewCellEditingStyle {
        if source == nil {
            return .None
        } else {
            return .Delete
        }
    }
```